examples/cpp/Makefile

```diff
@@ -0,0 +1,10 @@
+CXX=riscv32-unknown-elf-c++
```

Comment:
Do I need to update the CI AMI's to install this compiler / toolchain? If so what package on ubuntu 22.04 provides it?